### PR TITLE
Add ISPC to scripts

### PR DESCRIPTION
**Pull request by Vsevolod-Livinskij**

### CODE REVIEW

File: scripts/ispc-disp [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
```
Comment (dbabokin) [inline scripts/ispc-disp:1]: 1. License. you can use short pointer to the license, like this one: https://github.com/ospray/ospray/blob/master/scripts/files-to-md5.sh#L3
2. This is new file without a single line of comments or general description of what this file is doing.

File: scripts/ispc-proxy [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (dbabokin) [inline scripts/ispc-proxy:1]: 1. License
2. No comment/description

Comment (Vsevolod-Livinskij) [reply scripts/ispc-proxy:1]: Done

File: scripts/ispc-disp (hunk 1 above)
Comment (Vsevolod-Livinskij) [reply scripts/ispc-disp:1]: Done

**Pull request merged**